netlify.toml

```diff
@@ -2,3 +2,17 @@
   base = "site/"
   command = "jekyll build"
   publish = "site/_site"
+[build.processing]
+  skip_processing = false # -> Enable processing
+[build.processing.css]
+  bundle = true # -> Concatenate consecutive CSS files together to reduce HTTP requests.
+  minify = true # -> Run CSS through a minifier to reduce file size.
+[build.processing.js]
+  bundle = true # -> Concatenate consecutive JS files together to reduce HTTP requests.
```

Comment:
Same as above

Reply:
There are a few differences noticed on the [test site](https://deploy-preview-1815--projectcontour.netlify.com/guides/deploy-aws-nlb/)  that Netlify build for this PR which indicate that the additional post processing has an effect.

A basic navigation test across 5 pages on the site with caching enabled
Live site: 104 requests, 3.45MB transferred
Test site: 85 requests, 3.34MB transferred

The match height JS is not indemnified on the live site but is on the test site. Which is part of the goal, provide a second level of optimization without needing to add plugins to Jekyll or create and manage a pipeline to perform post build processing.

There is a limitation of how much if the JS can be minified as a number of scripts are pulled from external sources on page load and not part of the processing.